wordpress-mobile/WordPress-Android
# Issue/uc batch moderation

Part of #15101

This PR adds batch moderation to the comment list.

To test:
- In the comment list, long press on a comment to select it.
- Select one or multiple comments and perform various moderation actions on it.
- Confirm that the action mode, and confirmation dialogs when trashing or deleting comments work.

PR submission checklist:

Regressions:
- None, this is a feature branch, and the feature is behind the flag.
